### DISCUSSION TITLE
Right-click query menu to add/remove comments (--) from selected rows

### DISCUSSION THREAD

**Discussion by marc-ford-aptiv** [Sequel-Ace/Sequel-Ace#1494] [Ideas]
In the query editor, I often want to comment out rows of the query as I de-construct or debug a query.
Today, when you highlight some text in the query editor and right-click, there is a whole context menu full of copy, paste, transformations, etc.

Could we add a "Add Comment" and "Remove Comment" option?

The first would just add "-- " to beginning of any row highlighted.
The second would look for "-- " at the beginning of any selected lines and remove it.

P.S.  Protect for lines that have leading spaces.  If line has "         -- JOIN abc a on a.b = c.d", you still remove the "-- ".

**Comment by Jason-Morcos** [comment]
Probably possible, but is this mostly achievable already with the existing cmd+/ keyboard shortcut?
